bcbio/variation/coverage.py

```diff
@@ -208,12 +207,17 @@ def run_mosdepth(data, target_name, bed_file, per_base=False, quantize=None, thr
     bam_file = dd.get_align_bam(data) or dd.get_work_bam(data)
     work_dir = utils.safe_makedir(os.path.join(dd.get_work_dir(data), "coverage", dd.get_sample_name(data)))
     prefix = os.path.join(work_dir, "%s-%s" % (dd.get_sample_name(data), target_name))
+    thresholds = thresholds and LooseVersion(check_output(["mosdepth", "--version"]).split()[1].strip()) >= LooseVersion("0.2.0")
```

Comment:
We can assume that the user will have an up to date mosdepth so always apply the thresholds argument. The case we want to catch is when mosdepth has already been run with an older version (so won't have the thresholds file).